app/core/src/main/java/com/fsck/k9/job/K9JobManager.kt

```diff
@@ -0,0 +1,54 @@
+package com.fsck.k9.job
+
+import com.evernote.android.job.JobManager
+import com.fsck.k9.Preferences
+import timber.log.Timber
+
+class K9JobManager(
+        private val jobManager: JobManager,
+        private val jobCreator: K9JobCreator,
+        private val preferences: Preferences
+) {
+
+    init {
+        jobManager.addJobCreator(jobCreator)
+    }
+
+    fun scheduleAllMailJobs() {
+        Timber.v(TAG, "scheduling all jobs")
```

Comment:
Timber doesn't use manual log tags. This will treat `TAG` as format string and use "scheduling all jobs" as argument for the string formatting. Since there are no format specifiers in `TAG` the output will simply be "K9JobManager" (the contents of `TAG`).